Tasks/SSH/ssh.ts

```diff
@@ -55,6 +55,19 @@ async function run() {
 
         if(runOptions === 'commands') {
             commands = tl.getDelimitedInput('commands', '\n', true);
+        } else if (runOptions === 'inline') {
+            var inlineScript: string = tl.getInput('inline', true);
+            const scriptHeader:string = '#!';
+            if (inlineScript && !inlineScript.startsWith(scriptHeader)) {
+                inlineScript = '#!/bin/bash' + os.EOL + inlineScript;
```

Comment:
Can you output an informational message that says we're adding this to their script?